compiler/src/dmd/backend/arm/cod4.d

```diff
@@ -182,7 +154,7 @@ void cdeq(ref CGstate cg, ref CodeBuilder cdb,elem* e,ref regm_t pretregs)
             !(sz == 1 && e1.Voffset == 1)
            )
         {
-            if (varregm & XMMREGS)
+            if (0 && varregm & XMMREGS)
```

Comment:
why leave statically false branches in?

Reply:
A marker for where I will add the SIMD support.